setup.py

```diff
@@ -18,10 +18,12 @@
 requirements = [
         'pandas',
         'numpy',
+        'matplotlib',
         'rasterio',
         'pyyaml',
         'rasterstats>=0.17',
         'tqdm',
+        'pygeos',
```

Comment:
I wonder if we need pygeos, but perhaps that's a different issue.

Reply:
I notices it was enlisted in the requirements.txt, but not in here, so I just made sure both are the same.